FiveCalls/FiveCalls/AppState.swift

```diff
@@ -11,6 +11,7 @@ import CoreLocation
 import os
 
 class AppState: ObservableObject, ReduxState {
+    @Published var numberOfCalls: Int = 0
```

Comment:
A bit ambiguously named, since we count calls that you've made, calls on a particular issue, etc. `globalCallCount`?